ingest/src/DatabaseConnection.test.ts

```diff
@@ -6,7 +6,7 @@ import {
   DatabaseConnection,
 } from "./DatabaseConnection";
 import { PageStore, PersistedPage } from "./updatePages";
-import { ChunkStore, EmbeddedChunk } from "./updateChunks";
+import { ChunkStore } from "./updateChunks";
```

Comment:
in the Q&A server we call this content. can we change this to unify on name `content`?

Reply:
I think we should discuss that - what makes a "page" NOT "content"? I'm not married to "chunks" but we can probably come up with something descriptive/self-explanatory

Reply:
something like 'embedded content' could be more descriptive.

the idea w content and not chunks is if we at some future point also embed other forms of content like imgs or video

Reply:
I'm good with EmbeddedContent. Would do in subsequent PR.

Reply:
sgtm. that should also be updated in the Q&A server. can do 1 PR across both. can you make ticket or add to existing one like [DOCSP-30955](https://jira.mongodb.org/browse/DOCSP-30955)